src/OrleansSQLUtils/Storage/AdoNetInvariants.cs

```diff
@@ -6,6 +9,18 @@ namespace Orleans.SqlUtils
     public static class AdoNetInvariants
     {
         /// <summary>
+        /// A list of the supported invariants.
+        /// </summary>
+        public static ICollection<string> Invariants { get; } = new Collection<string>(new List<string>(new[]
```

Comment:
This isn't used in the code, it can be replaced with a method that creates this list via reflection on the type.